tests/test_readme.py

```diff
@@ -110,6 +110,21 @@ def test_string_without_content_type_setuppy(project) -> None:
     assert readme_path.read_text(encoding="utf-8").rstrip("\n") == long_description
 
 
+def test_file_with_space(project) -> None:
+    setup_cfg = """\
+[metadata]
+name = test-project
+version = 0.0.1
+long_description = file: README.rst
```

Comment:
Do we know if `setup.cfg` quietly strips trailing whitespace from strings?  I know it's an INI-file derivative, but not 100% certain on the semantics.

I'm confident the code will handle it anyway, so not overly concerned, but it might be helpful to add a similar test that makes sure.

Reply:
Oh, yeah good idea, although personally I wouldn't consider it a blocker for this PR. I just wanted to make sure there is _some_ test exercising the change, but additional tests for edge cases can always come later.

Reply:
Documented in #115, and we can figure out later what makes sense.